core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -1,8 +1,6 @@
 <script src="/assets/rich_text_components_definitions.js"></script>
 <script src="/assets/constants.js"></script>
-{% if not DEV_MODE %}
-  <script src="/assets/hashes.js"></script>
-{% endif %}
+<script ng-if="!DEV_MODE" ng-src="/assets/hashes.js"></script>
```

Comment:
Can you check if `ng-src` work with script tag? I am getting 404s with `bash script/start.sh --prod_env`.
Also on Travis as well:
https://travis-ci.org/oppia/oppia/jobs/417433278#L1404